src/index.js

```diff
@@ -1,17 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] As I can see you add your component inside `index.js` but in the requirements you should add it in your `app.js` so kindly create your App.js file and make it class component as well 🔥💯♻🚀that's very important to follow React flow because the index.js just work with our App and we put our components in app.js
![image](https://user-images.githubusercontent.com/58553711/205923258-cf05e943-1895-4602-8ff0-0bcf7c80db23.png)

Reply:
Thank you for your review🙂👍.I will start to work on issues